rplugin/python/nvim_ipy.py

```diff
@@ -265,7 +269,7 @@ def ipy_run(self, args):
             if p.get("source") == "page":
                 # TODO: if this is long, open separate window
                 self.append_outbuf(p['text'])
-
+        self.buf.name = self.base_buf_name + '({}s elapsed)'.format(time.time()-start_time) 
```

Comment:
using kernel-side `%time` is more reliable, because if one enqueues several commands (i e sends more requests before the last one is done) the timing will still be correct

> But maybe all that is needed is to be able to "decorate" each call to the kernel and display the output somewhere.

That should be possible with something like
`nnoremap <Leader>t :call IPyRun('%time '.getline('.'))<cr>`